.github/workflows/ci.yml

```diff
@@ -42,21 +57,13 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
-    - name: Set up Rust
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: nightly-2022-12-11
-        override: true
-        components: rustfmt
-    - name: Build
-      run: ./gradlew build
```

Comment:
I think installing Rust again and running the build again are unnecessary here? Running `./gradlew universalJar` further down does the main build as a prerequisite.